### DISCUSSION TITLE
Minor UX change: submission options

### DISCUSSION THREAD

**Discussion by nottheend** [firefly-iii/firefly-iii#7262] [General]
Hi, thanks for constantly inproving firefly-iii 👏🏻

In v6.0.5, there are 2 submission options introduced:
"Apply rules" and "fire webhooks" for the new transaction page.

Is it possible to move them above the "Submission" area with the "Submission" button?

Because I usually scroll down to the bottom without consciousness to skip many options and now I am not hitting the "Submit" button anymore 🫠

**Comment by JC5** [comment]
Thanks for the compliments! As for your question, sorry, people with the muscle memory the other way around would complain. 😅
